### DISCUSSION TITLE
Will Carbon be able to Use the win32 API (windows.h)?

### DISCUSSION THREAD

**Discussion by Icicles0** [carbon-language/carbon-lang#2797] [Getting started] [answered]
I'm just wondering Carbon will be able to make use of the windows API in the future.

**Comment by chandlerc** [comment, marked as answer]
Yes, eventually Carbon's C and C++ interop should allow use of those APIs.

That said, `windows.h` is an especially large and complex header and so I wouldn't expect it to be the first thing that starts to work.

**Reply by Icicles0** [reply]
Ok thank you!
